### PR TITLE
DW 5151 Deploy livy proxy

### PR DESCRIPTION
**Pull request by draagc-iw**

### CODE REVIEW

File: terraform/modules/livy-proxy/acm.tf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+resource "aws_acm_certificate" "pushgateway" {
```
Comment (IanUtley) [inline terraform/modules/livy-proxy/acm.tf:1]: pushgateway?

File: terraform/modules/livy-proxy/alb.tf [hunk 2]
```diff
@@ -0,0 +1,43 @@
+resource "aws_lb" "lb" {
+  name                             = "${var.name}-lb"
```
Comment (IanUtley) [inline terraform/modules/livy-proxy/alb.tf:2]: Is this name unique enough? Could we be overriding existing logs in the ELBLog directory for another load balancer?

Comment (draagc-iw) [reply terraform/modules/livy-proxy/alb.tf:2]: It's what we do for all our load balancers (and other resources), `var.name` is `livy-proxy` and we do not have any resources that use that name prefix.

File: terraform/modules/livy-proxy/ecs.tf [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      {
+        "containerPort": ${var.container_port}
+      }
+    ],
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "${aws_cloudwatch_log_group.livy-proxy.name}",
+        "awslogs-region": "${data.aws_region.current.name}",
+        "awslogs-stream-prefix": "container-internet-proxy"
```
Comment (IanUtley) [inline terraform/modules/livy-proxy/ecs.tf:25]: Why does the prefix mention internet-proxy?

File: terraform/modules/livy-proxy/outputs.tf [hunk 4]
```diff
@@ -0,0 +1,15 @@
+output "push_gateway_sg" {
```
Comment (IanUtley) [inline terraform/modules/livy-proxy/outputs.tf]: push-gateway again

File: terraform/modules/livy-proxy/ecs.tf (hunk 3 above)
Comment (draagc-iw) [reply terraform/modules/livy-proxy/ecs.tf:25]: Yep, my bad

**Review by IanUtley — APPROVED** [review]

**Review by caseyr232 — CHANGES_REQUESTED** [review]
Please don't merge this yet, need to check something from a security front

**Review by caseyr232 — APPROVED** [review]

**Pull request merged**